UPGRADING.rst

```diff
@@ -95,3 +95,20 @@ Overview of removed Graylog web interface settings:
 +---------------------------------+----------------------------------+--------------------+
 | ``web_tls_key_password``        | ``http_tls_key_password``        | Empty              |
 +---------------------------------+----------------------------------+--------------------+
+
+Plugins moved into the Graylog server
+=====================================
+
+In 3.0 we merged the following plugins into the server Graylog server repository:
+
+- `Beats Input <https://github.com/Graylog2/graylog-plugin-beats>`_
+- `CEF Input <https://github.com/Graylog2/graylog-plugin-cef>`_
+- `Collector Plugin <https://github.com/Graylog2/graylog-plugin-collector>`_
+- `Enterprise Integration Page <https://github.com/Graylog2/graylog-plugin-enterprise-integration>`_
+- `Map Widget <https://github.com/Graylog2/graylog-plugin-map-widget>`_
+- `NetFlow Input <https://github.com/Graylog2/graylog-plugin-netflow>`_
+- `Pipeline Processor <https://github.com/Graylog2/graylog-plugin-pipeline-processor>`_
+
+That means these plugins are not available as separate plugins anymore. If you manually update your Graylog installation (without using operating system packages), make sure to remove all old plugin files from the ``plugin/`` folder.
```

Comment:
Let's add a reference to the `plugin_dir` configuration setting as the plugins may not reside in a directory named `plugin`.